### ISSUE TITLE
Missing strings in v5 language file for v5.20.4

### ISSUE THREAD

**Issue by NagBoard** [richardwilkes/gcs_translations#4]
# Character Sheet sections

**Miscellaneous section (at the top)**
- Header of the section (the word `Miscellaneous` itself)
- Modified field (word `Modified`)

**Identity section (at the top)**
- Name field (word `Name`) 

**Encumbrance Table**
- Medium level of encumbrance in encumbrance table (word `Medium`)
- Move column header in encumbrance table (word `Move`)

**Weapon Table**
- Melee weapon column header in weapons table (word `Melee weapon`)
- Melee weapon usage column header in weapons table (word `Melee weapon usage`)
- Minimum ST of equip tooltip in weapons table
- Under parry and block columns when there is no parry or block provided by the equipment, word `No` is displayed in plain English

# Editors
**General Settings**
- Under calendar words `Gregorian` and  `Golarian` are in plain English

**Equipment editor**
- Name field in equipment table (word `Name`)

**Tool bar (inside editors)**
- Menu tooltip for menu button with three dashes icon (word `Menu` is in plain English when hovering on three dashes) 

**Fonts**
- Monospaced field name (word `Monospaced`)

# Menu bar
- Word `Minimize` under Window menu bar 
- Word `Menu keys...` under Settings menu bar
- Word `Move to other equipment` under Edit menu bar
- Word `Move to carried equipment` under Edit menu bar

**Comment by richardwilkes** [comment]
I'm not sure what file you're looking at, but the file attached to the release (https://github.com/richardwilkes/gcs/releases/download/v5.20.4/language.i18n) has the words in the list you mention above listed, at least for the first handful that I checked. These are actually plucked straight from the source code at build time, so should always be exactly what that binary is expecting.

**Comment by richardwilkes** [comment]
For example, the first word you mention, "Miscellaneous" is listed on line 2353 of the language.i18n file.

**Comment by NagBoard** [comment]
> For example, the first word you mention, "Miscellaneous" is listed on line 2353 of the language.i18n file.

Turned out I accidently duplicated around a hundred strings and that's why it was all messed up, funny enough almost all the "missing strings" were starting on M or N. 

Nevertheless, there are still several things which are really missing from the file, I Control+F'd them. Those are
- Words `Gregorian` and `Golarian`
- And there is the word `No` at line 2545, but somehow Parry and Block still are displayed as `No` instead of `Нет`

**Comment by NagBoard** [comment]
![image](https://github.com/richardwilkes/gcs_translations/assets/100212155/68585f5a-c1a7-46fd-a00f-81773d9ee25c)

**Comment by NagBoard** [comment]
Aside from those two I've found all the other strings as mentioned above.

**Comment by richardwilkes** [comment]
Anything that is stored in a data file will never be localized. `Gregorian` and `Golarion`, for example, are from the calendar data files. The `No` in the various weapon entries are likewise from data files and will not be localized.

**Comment by NagBoard** [comment]
Oh, I see. Then the issue resolved itself.
